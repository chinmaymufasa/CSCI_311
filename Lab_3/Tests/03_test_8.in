2
57
5
2
175
5
2
39
5
2
34
5
2
193
5
-1
