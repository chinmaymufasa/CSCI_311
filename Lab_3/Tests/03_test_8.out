57 
57 175 
39 57 175 
34 39 57 175 
34 39 57 175 193 
